grammars/css.cson

```diff
@@ -749,30 +1823,121 @@
     'beginCaptures':
       '0':
         'name': 'punctuation.definition.string.begin.css'
-    'end': '"'
+    'end': '"|(?<!\\\\)(?=$|\\n)'
     'endCaptures':
       '0':
         'name': 'punctuation.definition.string.end.css'
     'name': 'string.quoted.double.css'
     'patterns': [
       {
-        'match': '\\\\(\\h{1,6}|.)'
-        'name': 'constant.character.escape.css'
+        'begin': '(?:\\G|^)(?=(?:[^\\\\"]|\\\\.)+$)'
+        'end': '$'
+        'name': 'invalid.illegal.unclosed.string.css'
+        'patterns': [
+          {
+            'include': '#escapes'
+          }
+        ]
+      }
+      {
+        'include': '#escapes'
       }
     ]
   'string-single':
     'begin': '\''
     'beginCaptures':
       '0':
         'name': 'punctuation.definition.string.begin.css'
-    'end': '\''
+    'end': "'|(?<!\\\\)(?=$|\\n)"
     'endCaptures':
       '0':
         'name': 'punctuation.definition.string.end.css'
     'name': 'string.quoted.single.css'
     'patterns': [
       {
-        'match': '\\\\(\\h{1,6}|.)'
-        'name': 'constant.character.escape.css'
+        'begin': "(?:\\G|^)(?=(?:[^\\\\']|\\\\.)+$)"
+        'end': '$'
+        'name': 'invalid.illegal.unclosed.string.css'
+        'patterns': [
+          {
+            'include': '#escapes'
+          }
+        ]
+      }
+      {
+        'include': '#escapes'
+      }
+    ]
+  'tag-names':
```

Comment:
I'm fine with being wrong, would have preferred to get a less moody anwser but whatever. Please go on :)